radicle/src/cob/issue.rs

```diff
@@ -515,15 +536,21 @@ mod test {
 
         let assignee: ActorId = arbitrary::gen(1);
         let mut issue = issues
-            .create("My first issue", "Blah blah blah.", &[], &signer)
+            .create(
+                "My first issue",
+                "Blah blah blah.",
+                &[],
+                &[assignee],
+                &signer,
+            )
             .unwrap();
 
-        issue.assign(vec![assignee], &signer).unwrap();
-        issue.assign(vec![assignee], &signer).unwrap();
+        issue.assign([assignee], &signer).unwrap();
```

Comment:
This is no longer testing the same thing, I'd suggest having two different assignees, one of them is added in issue creation, and the other one is added *twice* after issue creation. Then we test that each is returned once.

Reply:
This should do the trick 6c53eeb